ui/apps/platform/src/Containers/Collections/CollectionsFormPage.tsx

```diff
@@ -59,7 +59,7 @@ function CollectionsFormPage({
                 {/* TODO - Handle UI for parse errors */}
             </>
         );
-    } else if (loading && !initialData) {
+    } else if (loading) {
```

Comment:
We need to hold off on rendering the form until loading is complete, even if we have a default value for `initialData`, since Formik will use this default data as state and ignore the request result once completed.